agent/src/mingwX64Main/kotlin/com/epam/drill/agent/configuration/DrillInstallationDir.kt

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2020 - 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.epam.drill.agent.configuration
+
+import kotlinx.cinterop.cstr
+import kotlinx.cinterop.invoke
+import kotlinx.cinterop.memScoped
+import kotlinx.cinterop.toKString
+import platform.posix.getenv
+import platform.posix.getpid
+import platform.posix.pclose
+import platform.posix.popen
+import io.ktor.utils.io.core.readText
+import io.ktor.utils.io.streams.Input
+
+internal actual fun drillInstallationDir() = run {
+    val isContainsAgentPath: (String) -> Boolean = { it.contains("-agentpath:") }
+    val fromEnv: () -> String? = { getenv("JAVA_TOOL_OPTIONS")?.toKString() }
+    val agentLine = fromEnv()?.takeIf(isContainsAgentPath) ?: fromWmic().takeIf(isContainsAgentPath)
+    val agentPath = Regex("-agentpath:(.+?)=.+").matchEntire(agentLine!!)!!.groups[1]!!.value

```

Comment:
1. This regex will fail if a user will pass only the path to the agent files without params ater `=`
e.g. `-agentpath:/data/agent/librill_agent.so`

This is a valid case, since we allow to pass config [via SYSTEM_CONFIG_PATH](https://github.com/Drill4J/java-agent/pull/124/files#diff-8fbddec29dee0ce54ba8e5bad63bcbef2048f12a3937519dd1e4f45eeec293d3L170) 

2. I suggest implement accepting path to config via System property (`-Dkey=val`) instead of SYSTEM_CONFIG_PATH env var
That will allow seamless Drill Java Agent installation - no changes required to AUT docker images
So it will work via JAVA_TOOL_OPTS completely:
```
image: my-aut:1.2.3
environment:
  JAVA_TOOL_OPTS="-agentpath=/agent/libdrill_agent.so -Ddrill.agent.config=/agent/drill.properties"
```

Reply:
edit:
1.altering AUT docker image isn't required since docker accepts env variables arbitrary names
2.system properties aren't available in native part, or, to be exact, require manual parsing